src/test/java/org/folio/list/service/ListServiceTest.java

```diff
@@ -389,20 +382,6 @@ void shouldCreateListWithDefaultFieldsIfFieldsNotProvided() {
     assertEquals(expectedFields, list.getFields());
   }
 
-  @Test
-  void createListShouldThrowErrorForUserMissingEntityTypePermissions() {
-    ListRequestDTO listRequestDto = TestDataFixture.getListRequestDTO();
-    when(entityTypeClient.getEntityType(listRequestDto.getEntityTypeId())).thenThrow(new FeignException.Unauthorized("[{\"User is missing permissions: [foo.bar]\"}]", mock(feign.Request.class), null, null));
-    assertThrows(InsufficientEntityTypePermissionsException.class, () -> listService.createList(listRequestDto));
-  }
-
-  @Test
-  void createListShouldThrowErrorForMissingEntityType() {
-    ListRequestDTO listRequestDto = TestDataFixture.getListRequestDTO();
-    when(entityTypeClient.getEntityType(listRequestDto.getEntityTypeId())).thenThrow(new FeignException.NotFound("Entity type not found", mock(feign.Request.class), null, null));
-    assertThrows(NotFoundException.class, () -> listService.createList(listRequestDto));
-  }
-
```

Comment:
These tests are not gone! The logic is moved to `EntityTypeClient`, and there's a new `EntityTypeClientTest`